patches/pytorch/pytorch_18_string_view.patch

```diff
@@ -0,0 +1,142 @@
+diff --git a/c10/util/string_view.h b/c10/util/string_view.h
```

Comment:
Maybe deactivate formatting for this patch?

Reply:
Yes, and I will resubmit the patch as another PR since it is mandatory for building torch 1.8 with cuda 11.3